src/System.Text.Json/experiments/Json.Net.Test/Json.Net.Tests/JsonParser.cs

```diff
@@ -0,0 +1,565 @@
+﻿using System;
+
+namespace Json.Net.Tests
+{
+    public struct JsonParser : IDisposable
```

Comment:
Remind me why you have this type here? i.e. why do you define the parser in tests project?